### PR TITLE
feat(docs): platform context menu

### PR DESCRIPTION
**Pull request by tknickman**
### Description

Add a vercel.com style context menu

Right click on the vercel logo in preview!

### CODE REVIEW

**Comment by JohnPhamous** [comment]
non-blocking: should the Vercel + the Turbo* logo be the trigger?

So everything in this box

CleanShot 2023-04-07 at 13 47 51@2x

File: docs/components/LogoContext/index.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import classNames from "classnames";
+import { VercelLogo } from "./icons";
+import { PRODUCT_MENU_ITEMS, PLATFORM_MENU_ITEMS } from "./items";
+import type { MenuItemProps } from "./types";
+import { MouseEvent } from "react";
+import { useTurboSite } from "../SiteSwitcher";
+
+function MenuDivider({ children, ...other }: { children: string }) {
+  return (
+    <div
```
Comment (JohnPhamous) [inline docs/components/LogoContext/index.tsx:13]: This should probably be a `h3` so a11y tools can use it as a landmark to jump between the sections.

**Comment by tknickman** [comment]
@JohnPhamous I did think about that, I wanted a way to turbo-ify it a bit more and that's a good suggestion. Maybe we could make the entire section the trigger, and then also include a direct "Copy as SVG" for whichever site (pack or repo) you're currently on?

File: docs/components/LogoContext/items.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        width="76"
+        height="65"
+        viewBox="0 0 76 65"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path d="M37.5274 0L75.0548 65H0L37.5274 0Z" fill="${
+          theme === "dark" ? "#ffffff" : "#000000"
+        }" />
+      </svg>;`
```
Comment (JohnPhamous) [inline docs/components/LogoContext/items.tsx:33]:
```suggestion
      </svg>`
```

Comment (JohnPhamous) [reply docs/components/LogoContext/items.tsx:33]: Not valid HTML with the semi

File: docs/components/LogoContext/items.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        `<svg xmlns="http://www.w3.org/2000/svg" fill=${
+          theme === "dark" ? "#ffffff" : "#000000"
+        } viewBox="0 0 284 65"><path d="M141.68 16.25c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.46 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zm117.14-14.5c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.45 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zm-39.03 3.5c0 6 3.92 10 10 10 4.12 0 7.21-1.87 8.8-4.92l7.68 4.43c-3.18 5.3-9.14 8.49-16.48 8.49-11.05 0-19-7.2-19-18s7.96-18 19-18c7.34 0 13.29 3.19 16.48 8.49l-7.68 4.43c-1.59-3.05-4.68-4.92-8.8-4.92-6.07 0-10 4-10 10zm82.48-29v46h-9v-46h9zM37.59.25l36.95 64H.64l36.95-64zm92.38 5l-27.71 48-27.71-48h10.39l17.32 30 17.32-30h10.39zm58.91 12v9.69c-1-.29-2.06-.49-3.2-.49-5.81 0-10 4-10 10v14.8h-9v-34h9v9.2c0-5.08 5.91-9.2 13.2-9.2z" /><!-- With love, the ▲ team --></svg>`
```
Comment (JohnPhamous) [inline docs/components/LogoContext/items.tsx:48]:
```suggestion
        `<svg xmlns="http://www.w3.org/2000/svg" fill="${
          theme === "dark" ? "#ffffff" : "#000000"
        }" viewBox="0 0 284 65"><path d="M141.68 16.25c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.46 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zm117.14-14.5c-11.04 0-19 7.2-19 18s8.96 18 20 18c6.67 0 12.55-2.64 16.19-7.09l-7.65-4.42c-2.02 2.21-5.09 3.5-8.54 3.5-4.79 0-8.86-2.5-10.37-6.5h28.02c.22-1.12.35-2.28.35-3.5 0-10.79-7.96-17.99-19-17.99zm-9.45 14.5c1.25-3.99 4.67-6.5 9.45-6.5 4.79 0 8.21 2.51 9.45 6.5h-18.9zm-39.03 3.5c0 6 3.92 10 10 10 4.12 0 7.21-1.87 8.8-4.92l7.68 4.43c-3.18 5.3-9.14 8.49-16.48 8.49-11.05 0-19-7.2-19-18s7.96-18 19-18c7.34 0 13.29 3.19 16.48 8.49l-7.68 4.43c-1.59-3.05-4.68-4.92-8.8-4.92-6.07 0-10 4-10 10zm82.48-29v46h-9v-46h9zM37.59.25l36.95 64H.64l36.95-64zm92.38 5l-27.71 48-27.71-48h10.39l17.32 30 17.32-30h10.39zm58.91 12v9.69c-1-.29-2.06-.49-3.2-.49-5.81 0-10 4-10 10v14.8h-9v-34h9v9.2c0-5.08 5.91-9.2 13.2-9.2z" /><!-- With love, the ▲ team --></svg>`
```

missing quotes, was invalid HTML

Comment (JohnPhamous) [reply docs/components/LogoContext/items.tsx:48]: To test these 2, copy on the website then try pasting into Figma

**Review by JohnPhamous — APPROVED** [review]

**Review by mehulkar — APPROVED** [review]

**Pull request merged**